eng/Packages.Data.props

```diff
@@ -81,7 +81,7 @@
 
     <!-- Azure SDK packages -->
     <PackageReference Update="Azure.Communication.Common" Version="1.0.1" />
-    <PackageReference Update="Azure.Core" Version="1.21.0" />
+    <PackageReference Update="Azure.Core" Version="1.22.0" />
```

Comment:
Did you intend to make this update as part of this PR? I think this will happen as part of shipping Core this month.

Reply:
yeah its a no op now, but needed to add it so that I could make the changes